platform/testing/end-to-end/cypress-migration-guide.md

```diff
@@ -140,32 +144,34 @@ cy.server();
 
 // Stub request to get enrollment status
 cy.route({
-  method: 'GET',
-  url: '/v0/health_care_applications/enrollment_status*',
+  method: "GET",
+  url: "/v0/health_care_applications/enrollment_status*",
   status: 404,
   response: {
     errors: [
       {
-        title: 'Record not found',
-        detail: 'The record identified by  could not be found',
-        code: '404',
-        status: '404',
+        title: "Record not found",
+        detail: "The record identified by  could not be found",
+        code: "404",
+        status: "404",
       },
     ],
   },
-}).as('getApplication');
+}).as("getApplication");
 
 // Stub request to submit application
-cy.route('POST', '/v0/health_care_applications', {
-  formSubmissionId: '123fake-submission-id-567',
-  timestamp: '2016-05-16',
-}).as('submitApplication');
+cy.route("POST", "/v0/health_care_applications", {
+  formSubmissionId: "123fake-submission-id-567",
+  timestamp: "2016-05-16",
+}).as("submitApplication");
 ```
+
 Based on the above stubs, whenever the browser makes a `GET /v0/health_care_applications/enrollment_status*` or a `POST /v0/health_care_applications` request, Cypress will automatically handle the request with the stubbed response.
 
 You can consider `cy.route()` to be the built-in Cypress equivalent of the Nightwatch custom command `mockData()`.
 
 **Our Cypress setup that's shared across all tests will automatically set up some default stubbed responses.**
+
 - This involves starting `cy.server()` and stubbing the `/v0/maintenance_windows` and `/v0/feature_toggles` endpoints to return empty arrays, so you don't have to explicitly do any of this in your tests.
 - **Why is this necessary?** Most apps make these requests to determine the availability of the app and render a loading indicator while waiting for the responses. Thus, without stubbing these endpoints, your tests will likely time out and fail before the app finishes loading due to the requests taking the same amount of time to time out. With a defined response, the requests are able to resolve immediately, and the test can proceed without delay.
 - **If you want different responses from these requests,** you can override the default stubs by calling `cy.route()` on the same endpoints.
```

Comment:
What do you think about linking the deep nested directories in line 175/181 instead of wrapping them in code blocks?